src/client/sandboxes/event/active-window.js

```diff
@@ -0,0 +1,61 @@
+import * as MessageSandbox from '../message';
+
+
+const WINDOW_ACTIVATED_EVENT   = 'windowActivated';
+const WINDOW_DEACTIVATED_EVENT = 'windowDeactivated';
+
+
+var currentWindow  = window;
+var isIFrameWindow = currentWindow !== currentWindow.top;
+var activeWindow   = !isIFrameWindow ? currentWindow.top : null;
+var isActive       = !isIFrameWindow;
+
+
+function notifyPrevActiveWindow () {
+    if (activeWindow.top) {
+        try {
+            MessageSandbox.sendServiceMsg({
+                cmd: WINDOW_DEACTIVATED_EVENT
+            }, activeWindow);
+        }
+        catch (err) {
+            //NOTE: the error appears in IE when the corresponding iframe is removed
+            return void 0;
+        }
+    }
+}
+
+export function init () {
```

Comment:
Since we have init() function I think it should be better to call this ActiveWindowTracker and make it class instead of set of functions (moreover it has internal state)